pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImplTest.java

```diff
@@ -382,6 +378,56 @@ private void checkOwnershipState(String broker, NamespaceBundle bundle)
         assertFalse(otherLoadManager.checkOwnershipAsync(Optional.empty(), bundle).get());
     }
 
+    @Test(timeOut = 30 * 1000)
+    public void testSplitBundleAdminAPI() throws Exception {
+        String namespace = "public/default";
+        String topic = "persistent://" + namespace + "/test-split";
+        admin.topics().createPartitionedTopic(topic, 10);
+        BundlesData bundles = admin.namespaces().getBundles(namespace);
+        int numBundles = bundles.getNumBundles();
+
+        String firstBundle = bundles.getBoundaries().get(0) + "_" + bundles.getBoundaries().get(1);
+
+        admin.namespaces().splitNamespaceBundle(namespace, firstBundle, true, null);
+
+        assertEquals(admin.namespaces().getBundles(namespace).getNumBundles(), numBundles + 1);
+
```

Comment:
Can we assert if the child bundle ranges exist too?

Reply:
Yes, updated.